api/projects/serializers.py

```diff
@@ -25,18 +29,20 @@ class Meta:
             "organisation",
             "hide_disabled_flags",
             "enable_dynamo_db",
-            "use_edge_identities",
+            "migration_status",
         )
 
-    def get_use_edge_identities(self, obj: Project) -> bool:
-        if settings.PROJECT_METADATA_TABLE_NAME_DYNAMO:
-            identity_migrator = IdentityMigrator(obj.id)
-            return identity_migrator.is_migration_done or (
-                settings.EDGE_RELEASE_DATETIME
-                and obj.created_date >= settings.EDGE_RELEASE_DATETIME
-            )
+    def get_migration_status(self, obj: Project) -> str:
+        if not settings.PROJECT_METADATA_TABLE_NAME_DYNAMO:
+            return ProjectIdentityMigrationStatus.MIGRATION_NOT_STARTED.value
+        if (
+            settings.EDGE_RELEASE_DATETIME
```

Comment:
We should add this as a property on the Project model if it doesn't already exist?